docs/reference/api.md

```diff
@@ -825,7 +838,52 @@ const verifySignatures = async (message, compositeSignatures) => {
 
 #### Examples
 
-Use cases include cryptographic login, message validation, verifiable credentials, and others.
+- [fcl-next-harness](https://github.com/onflow/fcl-next-harness)
+
+---
+## `AppUtils.verifyAccountProof`
+
+A method allowing applications to cryptographically prove that a user controls an on-chain account. During user authentication, some FCL compatible wallets will choose to support the FCL `account-proof` service. If a wallet chooses to support this service, and the user approves the signing of message data, they will return `account-proof` data and a signature(s) that can be used to prove a user controls an on-chain account. 
+See [proving-authentication](https://github.com/onflow/fcl-js/blob/master/docs/reference/proving-authentication.mdx) documentaion for more details.
+
+⚠️ `fcl.config.network`, `fcl.config.env` or options override is required to use this api. See [FCL Configuration](#configuration).
```

Comment:
Include twice? https://github.com/onflow/fcl-js/pull/1149/files#diff-370381df31c8d799afe0208a3b1f1c161b3f48ad3c5b5813e481c42f46161797R810

Reply:
sorry, what's this referring too?

Reply:
i think i see. verifyUserSignatures needs rewording to not reference proving account ownership. See change